components/Card/index.js

```diff
@@ -1,15 +1,30 @@
 import styled from "styled-components";
 
-export default function Card({ wish, why, price, howMuch, frequency }) {
+export default function Card({
+  what,
+  why,
+  price,
+  howMuch,
+  frequency,
+  id,
+  onDeleteCard,
+}) {
   return (
     <>
       <StyledCard>
         {" "}
-        <p>What: {wish}</p>
+        <div>What: {what}</div>
         <div>Why: {why}</div>
         <div>Price: {price}</div>
         <div>How Much: {howMuch}</div>
         <div>Frequency: {frequency}</div>
```

Comment:
You could better use `p` tags instead of `div` tags. Or you could create a list.